packages/contract/script/pay/DeployDaimoPayAcrossBridger.s.sol

```diff
@@ -146,39 +180,70 @@ contract DeployDaimoPayAcrossBridger is Script {
                 pctFee: 240000000000000, // 0.024%
                 flatFee: 90000 // 0.09 USDC
             });
-
-            // TODO: Add worldchain bridge route
-            revert("Add worldchain bridge route");
+            bridgeRoutes[5] = DaimoPayAcrossBridger.AcrossBridgeRoute({
+                bridgeTokenIn: LINEA_MAINNET_BRIDGED_USDC,
+                bridgeTokenOut: WORLDCHAIN_MAINNET_BRIDGED_USDC,
+                pctFee: 300000000000000, // 0.03%
+                flatFee: 120000 // 0.12 USDC
+            });
+            bridgeRoutes[6] = DaimoPayAcrossBridger.AcrossBridgeRoute({
+                bridgeTokenIn: _getDAIAddress(block.chainid),
+                bridgeTokenOut: BLAST_MAINNET_USDB,
+                pctFee: 900000000000000, // 0.09%
+                flatFee: 270000000000000000 // 0.27 DAI
+            });
         } else if (block.chainid == OP_MAINNET) {
-            chainIds = new uint256[](1);
-            bridgeRoutes = new DaimoPayAcrossBridger.AcrossBridgeRoute[](1);
+            chainIds = new uint256[](3);
+            bridgeRoutes = new DaimoPayAcrossBridger.AcrossBridgeRoute[](3);
 
             chainIds[0] = LINEA_MAINNET;
+            chainIds[1] = WORLDCHAIN_MAINNET;
+            chainIds[2] = BLAST_MAINNET;
 
             bridgeRoutes[0] = DaimoPayAcrossBridger.AcrossBridgeRoute({
                 bridgeTokenIn: _getUSDCAddress(block.chainid),
                 bridgeTokenOut: LINEA_MAINNET_BRIDGED_USDC,
                 pctFee: 720000000000000, // 0.072%
                 flatFee: 180000 // 0.18 USDC
             });
-
-            // TODO: Add worldchain bridge route
-            revert("Add worldchain bridge route");
+            bridgeRoutes[1] = DaimoPayAcrossBridger.AcrossBridgeRoute({
+                bridgeTokenIn: _getUSDCAddress(block.chainid),
+                bridgeTokenOut: WORLDCHAIN_MAINNET_BRIDGED_USDC,
+                pctFee: 300000000000000, // 0.03%
+                flatFee: 120000 // 0.12 USDC
+            });
+            bridgeRoutes[2] = DaimoPayAcrossBridger.AcrossBridgeRoute({
+                bridgeTokenIn: _getDAIAddress(block.chainid),
+                bridgeTokenOut: BLAST_MAINNET_USDB,
+                pctFee: 900000000000000, // 0.09%
+                flatFee: 270000000000000000 // 0.27 DAI
```

Comment:
might still be worth deduping partially given that the L2>L2 numbers seem to match, and just L1>L2 is different